libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -273,6 +274,22 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateCrossCloudMenu = (content) => {
+  const crossCloudMenuEl = content.querySelector('.cross-cloud-menu');
+  if (!crossCloudMenuEl) return;
+
+  const crossCloudMenuContent = toFragment`<div class="feds-crossCloudMenu"></div>`;
+
+  [...crossCloudMenuEl.querySelector('ul').children].forEach((el, index) => {
```

Comment:
Could you not simply select `ul a` instead of children?

Reply:
Changed 👍